### ISSUE TITLE
reproduce results

### ISSUE THREAD

**Issue by Fungungun** [huanghoujing/person-reid-triplet-loss-baseline#16]
Hi 

I train and test the Market1501 with the default parameters but only got 66% for CMC1 and 43.3% for mAP.

The precision in training is about 99.99%. Do you have any idea what might be wrong?

Cheers

**Comment by huanghoujing** [comment]
Hi, thanks for your attention! I guess the problem is in the version of python and pytorch. Do you use python 2.7 and pytorch 0.3?

**Comment by Fungungun** [comment]
Hi 

Thanks for your reply. 

Problem fixed now I got similar results as reported in this repo with more epochs of training. But it's still strange that since the precision are both around 99.99% and I don't know why the results are so different. Anyway thanks for your work.

**Comment by huanghoujing** [comment]
Glad to hear your results. The number of possible triplets for the training set is huge. So training longer helps the model to see more triplets, which may be useful, although training precision has reached 99%.
In this code, a total of 300 epochs seems satisfying, which is better than 150 epochs. How many epochs did you train that gives you 66% CMC1?

**Comment by Fungungun** [comment]
Can't remember. I think I just used the default 150 epochs. I just realized that I stopped and the whole training process in my first try. So I probably have tested with an checkpoint instead of the final trained model. Then it should be my problem. Thanks for your reply and are you on Wechat? If so please add me 382968762
